model/otel/metrics.yaml

```diff
@@ -206,6 +206,63 @@ groups:
       - ref: error.type
         examples: ["rejected", "timeout", "500", "java.net.UnknownHostException"]
 
+  - id: metric.otel.sdk.exporter.metric.inflight
+    type: metric
+    metric_name: otel.sdk.exporter.metric.inflight
```

Comment:
I think current name could be confusing since we generally refer to a "metric" as a thing that has multiple timeseries (e.g. [in proto](https://github.com/open-telemetry/opentelemetry-proto/blob/42319f8b5bf330f7c3dd4a097384f9f6d5467450/opentelemetry/proto/metrics/v1/metrics.proto#L187))

what do you think of

- `otel.sdk.exporter.metric_data_point.inflight`, or
- `otel.sdk.exporter.timeseries.inflight`

Reply:
Good point, I like `otel.sdk.exporter.metric_data_point.inflight`. This way the metric will still show up when searching for `metric` and therefore makes it more easily discoverable than `timeseries`.